samples/unit-tests/highcharts/constructors/demo.js

```diff
@@ -190,7 +190,7 @@ QUnit.test('Highmaps', function (assert) {
         }
     });
 
-    chart = new Highcharts.Map({
+    chart = new Highcharts.MapChart({
```

Comment:
I think we need an alias for this constructor. `new Highcharts.Map` was the documented way of constucting until v5 or so, to we risk pulling the carpet under some old implementations with this change. It should be quite simple to set it in the master file, `Highcharts.Map = MapChart`.

Reply:
Its already there.

Reply:
Okay, but then we need to keep the old constructor in some of the tests so that we have coverage.